lib/chef/resource/chef_client_config.rb

```diff
@@ -87,6 +87,17 @@ class ChefClientConfig < Chef::Resource
         ]
       end
       ```
+
+      **Report directly to data collector endpoint (normally automate), [reference link](https://docs.chef.io/automate/data_collection/#configure-chef-infra-client-to-use-the-data-collector-endpoint-in-chef-automate)**
```

Comment:
```suggestion
      **Report directly to data collector endpoint (Automate), [reference link](https://docs.chef.io/automate/data_collection/#configure-chef-infra-client-to-use-the-data-collector-endpoint-in-chef-automate)**
```

Reply:
@chef/docs-team what does this link need to look like when it goes into docs?

Reply:
I agree with all suggestions and will add them in one batch once we have a response from the doc team.

Reply:
How about this:


```suggestion
      **Report directly to the [Chef Automate data collector endpoint](/automate/data_collection/#configure-chef-infra-client-to-use-the-data-collector-endpoint-in-chef-automate).**
```